### PR TITLE
Show avatar placeholder in settings

### LINKED ISSUE #552: Show default avatar while real avatar is loading in Settings
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-102)

### PR DESCRIPTION
**Pull request by flozia**
This PR adds a placeholder to the settings view that is displayed until the user avatar is successfully loaded.

**Screenshot**
MacOS

Resolves #552

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-782)

### CODE REVIEW

**Comment by bakulf** [comment]
Can you rebase this PR? If @lesleyjanenorton is happy with it, I would like to see it merged in 2.5.

**Comment by lesleyjanenorton** [comment]
@bakulf almost! I will finish reviewing this today.

**Review by lesleyjanenorton — CHANGES_REQUESTED** [review]

File: src/ui/components/VPNPanel.qml [hunk 1]
```diff
@@ -29,54 +28,24 @@ Item {
...
-                    anchors.fill: parent
-                    radius: logoSize / 2
-                    visible: false
-                }
-
-                layer.effect: OpacityMask {
-                    maskSource: maskImage ? mask : undefined
-                }
-
+            VPNAvatar {
```
[the remark this answers is not in the document]
Comment (flozia) [reply src/ui/components/VPNPanel.qml:64]: `VPNAvatar` is now only visible in the Settings view and the mentioned issues should be fixed. @lesleyjanenorton

**Review by lesleyjanenorton — APPROVED** [review]
Woohoo! looks great - thank you @flozia ! 🤩
Screen Shot 2021-08-18 at 3 38 32 PM

**Pull request merged**